modules/nlu/src/backend/slots/slot-tagger.test.ts

```diff
@@ -124,6 +130,7 @@ describe('makeExtractedSlots', () => {
     const extractedSlots = makeExtractedSlots(testIntent, u, tagResults)
 
     expect(extractedSlots.length).toEqual(1)
+    expect(extractedSlots[0].slot.entity).toBeUndefined()
```

Comment:
shouldn't that be null ?

Reply:
entity is null only after passing trought `MapStepToOutput()` in the prediction pipeline.